### PR TITLE
Update fusion doc and add ONNX build flag to buildlb doc

### PR DESCRIPTION
**Pull request by indie**

### CODE REVIEW

File: doc/sphinx/source/fusion/index.rst [hunk 1]
```diff
@@ -5,38 +5,47 @@ Optimize Graphs
...
+#. Verify that the optimization pass performs correctly, with any or all expected 
+   transformations, with the ``NGRAPH_SERIALIZE_TRACING`` option, which 
+   serializes a graph in the `json` format after a pass.
+#. Measure and evaluate your performance improvements with ``NGRAPH_CPU_TRACING``, 
+   which produces timelines compatible with ``chrome://tracing``.
 
-Optimization passes can be programmed ahead of time if you know what your graph 
-will look like when it's ready to be executed, or the optimization passes can 
-be figured out manually with *Interpreter* mode on a stateless graph. 
+Optimization passes can be programmed ahead of time if you know or can predict
```
Comment (Krovatkin) [inline doc/sphinx/source/fusion/index.rst]: Optimizations can be experimented without using any backend by registering a pass with pass manager(`Manager`) and calling `run_passes` on a Function and then inspecting a transformed graph.
Interpreter isn't different from any other backend, so optimization strategies are hard-coded and can't be changed dynamically.

File: doc/sphinx/source/buildlb.rst [hunk 2]
```diff
@@ -85,12 +85,13 @@ The process documented here will work on Ubuntu\* 16.04 (LTS) or on Ubuntu
       $ mkdir build && cd build
 
 #. Generate the GNU Makefiles in the customary manner (from within the 
-   ``build`` directory). This command sets the target build location to
-   be ``~/ngraph_dist``, where it can be easily located.  
+   ``build`` directory). This command also builds the Library with ONNX support
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/buildlb.rst]: fixed

File: doc/sphinx/source/buildlb.rst [hunk 3]
```diff
@@ -157,14 +158,15 @@ The process documented here will work on CentOS 7.4.
       $ make && make install     
 
 #. Clone the `NervanaSystems` ``ngraph`` repo via HTTPS and use Cmake 3.4.3 to 
-   build nGraph Libraries to ``~/ngraph_dist``. 
+   build nGraph Libraries to ``~/ngraph_dist``. This command also builds the
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/buildlb.rst]: fixed

File: doc/sphinx/source/fusion/index.rst [hunk 4]
```diff
@@ -5,38 +5,47 @@ Optimize Graphs
...
+executed using any |InG| :doc:`backend <../programmable/index>`, has optimizations 
+inherent at the hardware level: superior runtime characteristics to increase 
+training performance or reduce inference latency.   
+
+There are several ways to describe what happens when we capture and translate 
+the framework's output of ops into an nGraph graph. :term:`Fusion` is the term 
+we shall use in our documentation; the action also can be described as: 
+*combining*, *folding*, *squashing*, *collapsing*, or *merging* of graph 
+functions. The most common use case is to *fuse* a subgraph from the function 
+graph into :doc:`one of the nGraph Core ops <../ops/index>`.
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/fusion/index.rst]: Okay, I'll delete that sentence since it doesn't quite follow.

**Pull request merged**